services/src/datasets/external/gbif.rs

```diff
@@ -174,7 +207,77 @@ impl GbifDataProvider {
 
         let limit = &i64::from(options.limit);
         let offset = &i64::from(options.offset);
-        let mut params: Vec<&(dyn ToSql + Sync)> = vec![&taxonrank, limit, offset];
+        let mut params: Vec<&(dyn ToSql + Sync)> = vec![limit, offset];
+        filters.iter().for_each(|(_, value)| params.push(value));
+        let rows = conn.query(&stmt, params.as_slice()).await?;
+
+        Ok(rows
+            .into_iter()
+            .map(|row| {
+                let canonicalname = row.get::<usize, String>(0);
+                let num_points = row.get::<usize, i64>(1);
+
+                CollectionItem::Layer(LayerListing {
+                    id: ProviderLayerId {
+                        provider_id: GBIF_PROVIDER_ID,
+                        layer_id: LayerId(taxonrank.clone() + "/" + canonicalname.as_str()),
+                    },
+                    name: canonicalname.clone(),
+                    description: format!("{num_points} occurrences"),
+                    properties: vec![],
+                })
+            })
+            .collect::<Vec<_>>())
+    }
+
+    async fn get_datasets_search_items(
```

Comment:
why is this function never used?

Reply:
Removed in https://github.com/geo-engine/geoengine/pull/908/commits/e23802633dc6b4ae701c3913186079f906872523 since https://github.com/geo-engine/geoengine/pull/908/commits/37f6862cc45f1941c5189369051b38a83c79c72b made it obsolete